debian/ubuntu-snappy.run-hooks.service

```diff
@@ -1,10 +1,10 @@
 [Unit]
-Description=Run snappy compatibility hooks
+Description=Run snappy security policy generation updates
```

Comment:
How about "Regenerate snappy security policies", to avoid three the 5-noun sentence?